app/models/user.rb

```diff
@@ -59,9 +58,6 @@ class User < ApplicationRecord
   before_validation :format_phone_number
   validates :phone_number, phone: true, allow_blank: true, format: { with: /\A\+1[0-9]{10}\z/ }
   has_many :assigned_tax_returns, class_name: "TaxReturn", foreign_key: :assigned_user_id
-  has_and_belongs_to_many :supported_organizations,
```

Comment:
@jenny-heath This is great! I didn't see a migration deleting this join table (UsersVitaPartner). Does it make sense to add a migration to delete that join table?

Reply:
🤦🏻‍♀️  definitely missed that...